CodeModel/CodeParsers/CSharp/CaDETClassMetricCalculator.cs

```diff
@@ -37,10 +37,127 @@ internal class CaDETClassMetricCalculator
                 [CaDETMetric.DIT] = CountInheritanceLevel(parsedClass),
                 [CaDETMetric.DCC] = CountClassCoupling(parsedClass),
                 [CaDETMetric.ATFD_10] = GetAccessToForeignDataDirectly(parsedClass),
-                [CaDETMetric.NIC] = CountInnerClasses(parsedClass)
+                [CaDETMetric.NIC] = CountInnerClasses(parsedClass),
+                [CaDETMetric.WOC] = CountWeightOfClass(parsedClass),
+                [CaDETMetric.NOPA] = CountPublicAttributes(parsedClass),
+                [CaDETMetric.NOPP] = CountPublicProperties(parsedClass.Members),
+                [CaDETMetric.WMCNAMM] = GetWMCOfNotAccessorOrMuttatorMethods(parsedClass),
+                [CaDETMetric.BUR] = GetBaseClassUsageRatio(parsedClass),
+                [CaDETMetric.BOvR] = GetBaseClassOverridingRatio(parsedClass),
             };
         }
 
+        private static double GetBaseClassOverridingRatio(CaDETClass parsedClass)
```

Comment:
Since this is the last calculated metric I would expect to find the method at the bottom of the whole class. Same goes for the rest of the methods, ordering is important.